daemon/mgr/image.go

```diff
@@ -62,13 +80,22 @@ func (mgr *ImageManager) PullImage(pctx context.Context, image, tag string, out
 		close(wait)
 	}()
 
-	if err := mgr.client.PullImage(ctx, image+":"+tag, stream); err != nil {
+	img, err := mgr.client.PullImage(ctx, image+":"+tag, stream)
```

Comment:
At this place, I am wondering that if we could directory search image cache to find existence of user-specified image, instead of calling containerd?

Reply:
Yes, I think so. We can reduce the dependence on containerd when pouchd is running. The cache is loaded from containerd, it must be reliable.